### DISCUSSION TITLE
临时WIFI是否可以增加一个固定密码

### DISCUSSION THREAD

**Discussion by ZhenZinian** [Mygod/VPNHotspot#373] [Ideas]
感谢作者开发了这么强大的app

自己的用例是手机上开了SSR后，给quest2连手机热点来下载商店内容

但是发现每次开启临时WIFI，热点名称和密码都会变，导致每次都需要重新输入

不知道是否可以为开启临时WIFI增加一个设置固定名称和密码的功能？

**Comment by Mygod** [comment]
Use repeater or system Wi-Fi hotspot instead.
